pkgs/by-name/ra/raycast/package.nix

```diff
@@ -3,20 +3,29 @@
   stdenvNoCC,
   fetchurl,
   writeShellApplication,
+  cacert,
   curl,
   jq,
-  common-updater-scripts,
+  nix,
   undmg,
 }:
 
+let
+  hashes = {
+    arm = "sha256-q3pX/mOl/u9KMcAfvXm4giYKjnTB903N1ibubvaO9Uw=";
+    x86_64 = "sha256-l61AVKx+aYmgnVK8d+by2pKiu1cIAueLipRjOzCvib4=";
+  };
+in
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "raycast";
-  version = "1.88.3";
+  version = "1.88.4";
+  isArm = stdenvNoCC.hostPlatform.isDarwin || stdenvNoCC.hostPlatform.isAarch64;
```

Comment:
Why we need `stdenvNoCC.hostPlatform.isDarwin` here?

Reply:
The idea is that even if `meta.platforms` isn't respected *(e.g someone's using `NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1`)*, `isArm` would still use the "correct" boolean